src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -862,7 +862,15 @@ def _unpack_custom_build(self):
 
     build_local_archive = os.path.join(self.build_dir,
                                        self.custom_binary_filename)
-    if not blobs.read_blob_to_disk(self.custom_binary_key, build_local_archive):
+    custom_builds_bucket = environment.get_value('CUSTOM_BUILDS_BUCKET')
```

Comment:
per the other (internal) CL: would it make more sense to make this a project.yaml config instead of an env var?

Reply:
Yup